srcpkgs/libyang/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'libyang'
+pkgname=libyang
+version=0.16r3
+revision=1
+_ver="0.16-r3"
+wrksrc="${pkgname}-${_ver}"
+build_style=cmake
+hostmakedepends="doxygen"
+makedepends="pcre-devel"
+short_desc="YANG data modeling language library"
+maintainer="Cameron Nemo <camerontnorman@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/CESNET/libyang"
+distfiles="https://github.com/CESNET/libyang/archive/v${_ver}.tar.gz"
+checksum=4745460dedc4ba17d8bcfc39ad9ba0d1b91bbe82b55b9417a090390909ba8ca5
+
+post_install() {
+	vlicense LICENSE
+}
+
+libyang-devel_package() {
+	short_desc+=" - development files"
+	depends="${srcpkgname}-${version}_${revision}"
```

Comment:
I have seen sourcepkg usually. Is srcpkgname something different or typo?

Reply:
probably a typo